crates/integration_tests/testdata/spark/provision.py

```diff
@@ -120,6 +120,14 @@
 spark.sql("ALTER TABLE rest.default.test_promote_column ALTER COLUMN foo TYPE bigint")
 spark.sql("INSERT INTO rest.default.test_promote_column VALUES (25)")
 
+#  Create a table, and do some evolution on a partition column
+spark.sql("CREATE OR REPLACE TABLE rest.default.test_promote_partition_column (foo int, bar float, baz decimal(4, 2)) USING iceberg")
```

Comment:
```suggestion
spark.sql("CREATE OR REPLACE TABLE rest.default.test_promote_partition_column (foo int, bar float, baz decimal(4, 2)) USING iceberg PARTITIONED BY (foo)")
```

the initial version had PARTITIONED BY 
https://github.com/apache/iceberg-rust/pull/1334/commits/5fa9ef8a0f6ebd907bab30ab75193561e61530d4#diff-346411a7e9377582f5b68c60ddcf61e24b474d5c4a686f9c7195e576c9c9168dR118

Reply:
Copy-paste, thanks! :D